libs/ui/layout/src/lib/cards-subsections/cards-subsections.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="flex flex-row items-center justify-between">
+  <div class="flex flex-row items-center gap-x-4">
+    <p
+      class="font-title text-[28px] font-medium text-title text-center sm:text-left"
+      [translate]="title"
+    ></p>
+    <gn-ui-previous-next-buttons
+      *ngIf="paginableElement?.pagesCount > 1"
+      [listComponent]="paginableElement"
+    ></gn-ui-previous-next-buttons>
+  </div>
+  <div id="filters-container" class="flex items-center gap-x-4">
+    <ng-content select="[slot=filters]"></ng-content>
+  </div>
```

Comment:
same thing, this should stay in the app I think; this is too much responsibility for this component; this being said, a card-subsections component could go in apps/datahub